src/app/screens/ledger/confirmLedgerTransaction/index.tsx

```diff
@@ -281,9 +204,9 @@ function ConfirmLedgerTransaction(): JSX.Element {
         return;
       }
 
-      const accountId = selectedAccount.deviceAccountIndex;
+      const addressIndex = selectedAccount.deviceAccountIndex;
```

Comment:
do we rename the `deviceAccountIndex` attribute, or does it still make sense as `deviceAccountIndex`?

Reply:
I think that it still makes sense to be named `deviceAccountIndex` since it starts with zero for each new device